Source/ASDisplayNode.mm

```diff
@@ -729,6 +729,8 @@ - (BOOL)_isNodeLoaded
 
 - (UIView *)view
```

Comment:
@nguyenhuy I changed the implementations of `view` and `layer`. Can you please take another look over it. Not sure yet though if we should add the main thread assertion on top.